packages/entities/entities-routes/vite.config.ts

```diff
@@ -23,6 +23,14 @@ const config = mergeConfig(sharedViteConfig, defineConfig({
         '@kong-ui-public/expressions/dist/style.css',
         'lodash.isequal',
       ],
+      output: {
+        // Provide global variables to use in the UMD build for externalized deps
+        globals: {
+          '@kong-ui-public/expressions': '@kong-ui-public/expressions',
+          '@kong-ui-public/expressions/dist/style.css': '@kong-ui-public/expressions/dist/style.css',
+          'lodash.isequal': 'lodash.isequal',
```

Comment:
You don't need `globals` for explicit style paths as they aren't imported via JS.

For the package names (the ones that we own) the format changes for the exposed global like this: 
```suggestion
          '@kong-ui-public/expressions': 'kong-ui-public-expressions',
          'lodash.isequal': 'lodash.isequal',
```

Here's an example of how the exposed global string is generated: https://github.com/Kong/public-ui-components/blob/6c9bb045f8e52a490333a1ab9b0066845aaefff1/packages/entities/entities-certificates/vite.config.ts#L15